web3/module.py

```diff
@@ -99,3 +99,5 @@ def __init__(self, web3: "Web3") -> None:
             self.retrieve_caller_fn = retrieve_async_method_call_fn(web3, self)
         else:
             self.retrieve_caller_fn = retrieve_blocking_method_call_fn(web3, self)
+        # TODO - remove once web3 calls are moved to middleware
+        self.web3 = web3
```

Comment:
Context would be appreciated here too :pray:

Reply:
For some reason, moving the child modules to use ModuleV2 doesn't allow them to attach them correctly. So modules like the `ParityPersonal` and `ParityShh` module  won't work without this here. I'll timebox digging in one more time to see if I can figure it out and let you know what I find!